be/src/services/entity.service.ts

```diff
@@ -19,16 +21,20 @@ export class EntityService {
 
     // TODO remove rule when implemented
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    getEntities(vocabID: string, filter: unknown): Promise<unknown> {
-        throw new KnowledgeError(501, 'Not Implemented', 'Operation not implemented');
+    async getEntities(vocabID: string | ObjectId, query: ListQueryModel, entityID?: string | ObjectId): Promise<ListingResult<Entity>> {
+        const {options, filter} = this.transformToMongoDBFilterOption(vocabID, query, entityID);
+        // @ts-ignore
+        const entities: Entity[] = await this.collection.find(filter, options).toArray() as Entity[];
+        return {
+            offset: query.offset,
+            rows: entities.length,
+            totalItems: 0, //TODO
+            items: entities
+        }
     }
 
-    // TODO remove rule when implemented
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    getEntity(vocabID: string, entityID: string): Promise<unknown> {
-        throw new KnowledgeError(501, 'Not Implemented', 'Operation not implemented',
-          {id:'dummyEntityErrorPayload'}
-        );
+    getEntity(vocabID: string | ObjectId, entityID: string | ObjectId): Promise<Entity> {
+        return this.collection.findOne({_id: new ObjectId(entityID), vocabulary: new ObjectId(vocabID)}).then(x => <Entity>x);
```

Comment:
Please, also check that the provided `vocabularyId` exists as a Vocabulary in the DB.

Reply:
GET entity + entities works, search functionality is excluded and will be programmed in another branch